### ISSUE TITLE
disable doubleenter if you make an emoji at the end of a line

### ISSUE THREAD

**Issue by senthai** [RocketChat/Rocket.Chat#2469]
If you enter an emoji at the end of the line someone has to press double enter to send.

Perhaps it is possible to send a message by pressing enter only once?

**Comment by engelgabriel** [comment]
I dont think there is a way around, the first enter is to autocomplete the emoji.. you can hit space or tab too.

**Comment by senthai** [comment]
is it possible to deactivate the autocomplete?

**Comment by engelgabriel** [comment]
Not at this moment, but you can open another issue requesting that feature or you can do a pull request.
